Include/cpython/frameobject.h

```diff
@@ -4,50 +4,17 @@
 #  error "this header file must not be included directly"
 #endif
 
-/* These values are chosen so that the inline functions below all
- * compare f_state to zero.
- */
-enum _framestate {
-    FRAME_CREATED = -2,
-    FRAME_SUSPENDED = -1,
-    FRAME_EXECUTING = 0,
-    FRAME_RETURNED = 1,
-    FRAME_UNWINDING = 2,
-    FRAME_RAISED = 3,
-    FRAME_CLEARED = 4
-};
-
-typedef signed char PyFrameState;
-
 struct _frame {
     PyObject_HEAD
     struct _frame *f_back;      /* previous frame, or NULL */
```

Comment:
Hmm can we remove this field and instead access this from `_PyFrame->previous->frame_obj` ?

Reply:
Debuggers tend to use this field quite a lot so I would advise against it.

Reply:
I forgot that frame.f_back is commonly used in Python too. Thanks for reminding me.

Reply:
We can't remove it because the the `previous` field is meaningless after a call returns, but `f_back` must still be valid.